woocommerce/woocommerce-gateway-stripe
# feat: automatic capture when enabling UPE

# Changes proposed in this Pull Request:

Adds a checkbox on the UPE onboarding wizard to enable/disable "manual" capture, if "manual" capture is enabled in the first place.

Helps with https://github.com/woocommerce/woocommerce-gateway-stripe/issues/1874 - splitting into multiple PRs

When "Automatic" capture is enabled, the checkbox is not displayed.

With "automatic" capture in the settings (no checkbox is displayed, user can continue to the next step):
![2021-10-04 15 40 40](https://user-images.githubusercontent.com/273592/135921868-c57233ce-764c-4624-a44b-1724bf850469.gif)



With "manual" capture in the settings:
![2021-10-04 15 28 44](https://user-images.githubusercontent.com/273592/135920237-34ea7418-3896-4b87-ac64-e999f082dc28.gif)

# Testing instructions
- Ensure you have the `_wcstripe_feature_upe_settings` feature flag enabled
- Ensure you have the UPE flag disabled
- Go to http://localhost:8082/wp-admin/admin.php?page=wc_stripe-onboarding_wizard
- If you have "manual" capture in the settings, the checkbox is displayed and you can't proceed until you confirmed to disable "manual" capture
